include/coap3/coap_debug.h

```diff
@@ -48,25 +48,27 @@ typedef short coap_log_t;
     with LOG_CIPHERS added. */
 
 #if !defined(RIOT_VERSION)
-typedef enum {
-  LOG_EMERG=0, /**< Emergency */
-  LOG_ALERT,   /**< Alert */
-  LOG_CRIT,    /**< Critical */
-  LOG_ERR,     /**< Error */
-  LOG_WARNING, /**< Warning */
-  LOG_NOTICE,  /**< Notice */
-  LOG_INFO,    /**< Information */
-  LOG_DEBUG,   /**< Debug */
-  COAP_LOG_CIPHERS=LOG_DEBUG+2 /**< CipherInfo */
-} coap_log_t;
+typedef int coap_log_t;
```

Comment:
Now that `coap_log_t` is always defined as `int`, it can be put outside the `#ifdef`s.

Reply:
FIxed